packages/core/contracts/financial-templates/funding-rate-store/implementation/FundingRateStore.sol

```diff
@@ -301,6 +417,34 @@ contract FundingRateStore is FundingRateStoreInterface, Testable, Lockable {
         store.payOracleFeesErc20(address(collateralCurrency), amount);
     }
 
+    function _calculateProposalRewardPct(
```

Comment:
This method returns reward amounts as:
- `reward % per second` * `seconds elapsed` * `% change in proposed funding rate from current rate`